estuary-cdk/estuary_cdk/http.py

```diff
@@ -21,6 +22,30 @@
 
 DEFAULT_AUTHORIZATION_HEADER = "Authorization"
 
+StreamedObject = TypeVar("StreamedObject", bound=BaseModel)
+
+class _AsyncStreamWrapper:
```

Comment:
This feels like it was a lot more difficult than it needed to be. `ijson.items` needs something "file-like" with a `read` method. `aiohttp` doesn't give us anything quite like that (not even its raw `content` seemed to work). So this adapter is the only way I could get things working with the byte stream coming from `aiohttp`.